### DISCUSSION TITLE
which account does the sniping & joining?

### DISCUSSION THREAD

**Discussion by yungcartionthachair** [marioparaschiv/nitro-sniper#333] [Q&A]
i want a sniper acc that redeems the nitro on my main, I dont wanna risk my main getting banned. where is the feeding config like mentioned in the features?

edit: i see the mode function under the tokens but will it still apply nitro to my main if im running it in alt mode? there should be a nitro priority function that allows you to snipe on one account but apply nitro to another account first.

**Comment by MrD4rio** [comment]
The nitro gets applied on the main token, even if it gets sniped from one of the alt accounts.
